CODEOWNERS

```diff
@@ -2,9 +2,11 @@
 # Each line is a file pattern followed by one or more owners.
 
 # These owners will be the default owners for everything in the repo.
-*       @microsoft/dynamics-smb
+* @microsoft/dynamics-smb
 
 # AL-Go files are owned by Engineering Systems
+/Modules/DevTools/.AL-Go @microsoft/dynamics-smb-engineering-systems
```

Comment:
Does not follow the current structure.

Reply:
Fix CODEOWNERS to match all .AL-Go folders in the repo.
Source https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

```
# In this example, @octocat owns any file in an apps directory
# anywhere in your repository.
apps/ @octocat
```